app/src/main/java/com/kodeco/android/coordplot/CountryApp/UI/CountryInfoScreen.kt

```diff
@@ -0,0 +1,68 @@
+package com.kodeco.android.coordplot.CountryApp.UI
+
+import android.util.Log
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.kodeco.android.coordplot.CountryApp.Model.Country
+import com.kodeco.android.coordplot.CountryApp.Networking.CountryApiObject.retrofitService
+
+var currentstatus:Boolean=false
+// TODO fill out CountryInfoScreen
+@Composable
+fun CountryInfoScreen( status:Boolean) {
+
+    //var logservice:List<Country> = remember {mutableStateListOf<Country>()}
+    var logservice:List<Country> by remember {
+        mutableStateOf(listOf<Country>())
+    }
+    currentstatus=status
+    if(status){
+        LaunchedEffect(Unit){
+            try {
+                // some code that can cause an exception.
+                logservice= retrofitService.getCountries().body()?: listOf<Country>()
```

Comment:
Here you can also check if the api response is successful or not. Sometimes the API can throw unexpected errors so we should also handle them gracefully.